src/pages/docs/platform/architecture/infrastructure-operations.mdx

```diff
@@ -0,0 +1,206 @@
+
+## Operational management
+
+Ably's operational management combines skilled personnel, well-defined processes, and continuous improvement to maintain service reliability.
+
+Ably has a permanent Software Reliability Engineering (SRE) team that manages the day-to-day operation of its services. This team is responsible for maintaining the health and performance of the platform, responding to incidents, and implementing operational improvements.
+
+The SRE team maintains a 24x7 on-call rotation, ensuring that skilled engineers are always available to respond to issues. In addition to the primary SRE rotation, other teams responsible for specific platform roles also operate on-call rotations, providing specialized expertise when needed. When alerts are triggered, they page the relevant on-call engineer based on the affected component or service.
+
+On-call engineers lead incident response and resolution, following established procedures to diagnose issues, implement mitigations, and coordinate broader responses when necessary. Beyond reactive incident response, the team continuously monitors system health and performance, identifying potential issues before they become critical.
+
+A key responsibility of the SRE team is to identify and implement improvements to operational practices. By analyzing past incidents and system behavior, they drive ongoing enhancements to monitoring, automation, and response procedures.
+
+### On-call processes
+
+Ably has implemented comprehensive on-call procedures designed to minimize both mean time to detection (MTTD) and mean time to resolution (MTTR) for service issues.
+
+Each alert has associated documentation explaining the issue, its severity, troubleshooting steps, and remediation actions. These playbooks provide on-call engineers with the information they need to respond effectively, even for unfamiliar components or rare failure modes.
+
+All pages are automated based on monitoring and alerting thresholds. Engineers aren't expected to watch dashboards, reducing the risk of human error or attention lapses in detecting issues. The automated paging system is regularly tested to ensure reliability, including routine test pages when engineers begin their on-call shift.
+
+For managing larger incidents, Ably maintains a defined incident management framework with specific roles and standardized procedures. This structure ensures clear communication, appropriate escalation, and effective coordination during complex or high-impact issues.
+
+These practices ensure that on-call engineers can respond efficiently to incidents with the information and support they need, reducing the stress of on-call duties while improving response effectiveness.
+
+## Fault tolerance and service continuity
```

Comment:
Worth mentioning status page in one of these sections?

Reply:
sure, added above

Reply:
b77ec120095617b99a5b43fb61dc82731224f02b